kitchen.yml

```diff
@@ -1,9 +1,10 @@
 driver:
   name: vagrant
+  chef_version: <%= ENV['CHEF_VERSION'] || 'current' %>
+  env: [ CHEF_LICENSE=accept ]
```

Comment:
I think we need to add the yamllint file to this repo, it'll fix these warnings.

Reply:
already has one